tests/help.js

```diff
@@ -1,14 +1,8 @@
 const chai = require('chai');
 const sinonChai = require('sinon-chai');
-const { JSDOM } = require('jsdom');
```

Comment:
do we still need JsDOM or can we remove the dependency?

Reply:
Ah yes 👍